### DISCUSSION TITLE
Grid View for tabs

### DISCUSSION THREAD

**Discussion by ghost** [bromite/bromite#1060] [Q&A]
Open tabs are seen in a grid like manner, is it possible to get back to the normal view ? I have disabled the chrome flag as well but no change! 
![Screenshot_20210413-112729_Bromite](https://user-images.githubusercontent.com/82433886/114503649-46d86880-9c4b-11eb-9f3f-8dddc0e26acb.jpg)

**Comment by hvjufg** [comment]
Noticed that

**Reply by ghost** [reply]
any fix?

**Reply by ghost** [reply]
Maybe is the chrome://flag#enable-tab-grid-layout?
Try to change to "Disabled" and see if helps.

**Reply by lucaslira95** [reply]
tried too, not working

**Reply by ghost** [reply]
Strange, I have Ungoogled Chromium (Which, is relatable to Bromite) and at first launch, the tabs are normal (Pile of tabs).

I set up the flag that I mentioned, and works fine. Now, I turned off to "Default" and worked fine too.

I'll download Bromite, open (not change anything other then the flag) and comment here.

**Reply by lucaslira95** [reply]
Now strange for me, because I let it disabled for some time and now the changes decided to work

**Reply by ghost** [reply]
So, on Bromite, seems like the grid layout is applied when the app is opened, even when the flag is set to "Default". I change to "Disabled", but not changed anything, even when I closed the app two times.

**Reply by lucaslira95** [reply]
Well, I came back to tabbed layout, closed app, cleaned app cache in android settings, and changes apply faster now :)

**Reply by ghost** [reply]
> Well, I came back to tabbed layout, closed app, cleaned app cache in android settings, and changes apply faster now :)

so it's back to normal for you now?

**Reply by lucaslira95** [reply]
yup
